src/Anonymization/Anonymizer/AnonymizerRegistry.php

```diff
@@ -4,134 +4,233 @@
 
 namespace MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer;
 
-use MakinaCorpus\DbToolsBundle\Attribute\AsAnonymizer;
+use Composer\InstalledVersions;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\AddressAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\ConstantAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\DateAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\EmailAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\FirstNameAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\FloatAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\IbanBicAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\IntegerAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\LastNameAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\LoremIpsumAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\Md5Anonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\NullAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\PasswordAnonymizer;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core\StringAnonymizer;
 
 class AnonymizerRegistry
 {
+    // Core anonymizers list is hardcoded.
+    private static $coreAnonymizers = [
+        AddressAnonymizer::class,
+        ConstantAnonymizer::class,
+        DateAnonymizer::class,
+        EmailAnonymizer::class,
+        FirstNameAnonymizer::class,
+        FloatAnonymizer::class,
+        IbanBicAnonymizer::class,
+        IntegerAnonymizer::class,
+        LastNameAnonymizer::class,
+        LoremIpsumAnonymizer::class,
+        Md5Anonymizer::class,
+        NullAnonymizer::class,
+        PasswordAnonymizer::class,
+        StringAnonymizer::class,
+    ];
+
     private ?array $anonymizers = null;
     private array $paths = [];
 
-    public function __construct(
-        private ?string $projectDir = null,
-        ?array $paths = null
-    ) {
-        $this->addPath($paths ?? [__DIR__]);
-        $this->locatePacks();
+    public function __construct(?array $paths = null)
+    {
+        $this->addPath($paths ?? []);
     }
 
-    public function addPath(array $paths): void
+    /**
+     * Add path in which to lookup for anonymizers.
+     */
+    public function addPath(array|string $paths): void
```

Comment:
I don't get why this has to be changed ? Is this not the exact same thing ?

I think we can keep the original signature

Reply:
I kept `array` for backward compatibility, but in `locatePacks` I simply add strings.